
<launch>

  <arg name="limb"     default="right"/>
  <arg name="method"   default="color"/>

  <node pkg="baxter_interface" name="joint_trajectory_action_server" type="joint_trajectory_action_server.py" />
  <include file="$(find baxter_moveit_config)/launch/demo_baxter.launch"/>

  <param name="robot_description" textfile="$(find baxter_description)/urdf/baxter.urdf"/>

  <param name="object_tracker/config_folder" value="$(find baxter_demos)/config/"/>

  <node pkg="baxter_demos" name="object_finder_3d" type="object_finder_3d" required="true">
  </node>
  
  <node pkg="baxter_demos" name="servo_to_object" type="stackit.py"
args="--limb $(arg limb) --visualize true" required="true">
    </node>



</launch>
